### PR TITLE
Draft Test PR - For CI Job Arrow Module 

### PR DESCRIPTION
**Pull request by lithinwxd**
## Contributor checklist

- [ ] Please make sure your submission complies with our [development](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#development), [formatting](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#formatting), [commit message](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#commit-formatting-and-pull-requests), and [attribution guidelines](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#attribution).
- [ ] PR description addresses the issue accurately and concisely.  If the change is non-trivial, a GitHub Issue is referenced.
- [ ] Documented new properties (with its default value), SQL syntax, functions, or other functionality.
- [ ] If release notes are required, they follow the [release notes guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines).
- [ ] Adequate tests were added if applicable.
- [ ] CI passed.

## Release Notes
Please follow [release notes guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines) and fill in the release notes below.

```
== RELEASE NOTES ==

General Changes
* ... :pr:`12345`
* ... :pr:`12345`

Hive Connector Changes
* ... :pr:`12345`
* ... :pr:`12345`
```

If release note is NOT required, use:

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-docs/src/main/sphinx/connector/base-arrow-flight.rst [hunk 1]
```diff
@@ -0,0 +1,96 @@
+
+======================
+Arrow Flight Connector
+======================
+This connector allows querying multiple data sources that are supported by an Arrow Flight server.
+Apache Arrow enhances performance and efficiency in data-intensive applications through its columnar memory layout, zero-copy reads, vectorized execution, cross-language interoperability, rich data type support, and optimization for modern hardware. These features collectively reduce overhead, improve data processing speeds, and facilitate seamless data exchange between different systems and languages.
+
+Getting Started with base-arrow-module: Essential Abstract Methods for Developers
+---------------------------------------------------------------------------------
+To use the base-arrow-module, you need to implement certain abstract methods that are specific to your use case. Below are the required classes and their purposes:
```
Comment (steveburnett) [inline presto-docs/src/main/sphinx/connector/base-arrow-flight.rst:10]:
```suggestion
To use the base-arrow-module, implement the following list of required abstract methods:
```
I'm not sure I understand "certain abstract methods that are specific to your use case", when all of the following list of classes is described as required. Let me know if my suggestion does not correctly describe the situation, or if I left out something that must be added.

File: presto-docs/src/main/sphinx/connector/base-arrow-flight.rst [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+* ``ArrowFlightClientHandler.java``
+  This class is responsible for initializing the Flight client and retrieving Flight information from the Flight server. To authenticate the Flight server, you must implement the abstract method ``getCallOptions`` in ArrowFlightClientHandler, which returns the ``CredentialCallOption`` specific to your Flight server.
```
Comment (steveburnett) [inline presto-docs/src/main/sphinx/connector/base-arrow-flight.rst:13]:
```suggestion
  This class is responsible for initializing the Flight client and retrieving Flight information from the Flight server. To authenticate the Flight server, implement the abstract method ``getCallOptions`` in ArrowFlightClientHandler, which returns the ``CredentialCallOption`` specific to your Flight server.
```

File: presto-docs/src/main/sphinx/connector/base-arrow-flight.rst [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+If you used a different name for your catalog properties file, use
+that catalog name instead of ``arrowmariadb`` in the above examples.
+
+
+Flight Connector Limitations
+----------------------------
+
+* SELECT and DESCRIBE queries are supported by this connector template. Implementing modules can add support for additional features.
+
+* Flight connector can query against only those datasources which are supported by the Flight server.
```
Comment (steveburnett) [inline presto-docs/src/main/sphinx/connector/base-arrow-flight.rst:94]: Can you add a link to what datasources are supported by the Flight server?

File: presto-docs/src/main/sphinx/connector/base-arrow-flight.rst [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+* The user should have the Flight server running for the Flight connector to work.
```
Comment (steveburnett) [inline presto-docs/src/main/sphinx/connector/base-arrow-flight.rst:96]:
```suggestion
* The Flight server must be running for the Flight connector to work.
```

File: presto-docs/src/main/sphinx/connector/base-arrow-flight.rst [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+``arrow-flight.server.port``                Flight server port
+``arrow-flight.server-ssl-certificate``     Pass ssl certificate
+``arrow-flight.server.verify``              To verify server
+``arrow-flight.server-ssl-enabled``         Port is ssl enabled
+========================================== ==============================================================
+
+Querying Arrow-Flight
+---------------------
+
+The Flight connector provides schema for each supported *database*.
```
Comment (steveburnett) [inline presto-docs/src/main/sphinx/connector/base-arrow-flight.rst:64]:
```suggestion
The Flight connector provides schema for each configured *catalog*.
```
Should `database` be `catalog` in this section?

File: presto-docs/src/main/sphinx/connector/base-arrow-flight.rst [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+  Extend the ArrowAbstractSplitManager class to implement the Arrow Flight request, defining the Arrow split.
+
+* ``ArrowPlugin.java``
+  Register your connector name by extending the ArrowPlugin class.
+
+* ``ArrowFlightRequest``
+  The ``getCommand`` method in the ``ArrowFlightRequest`` interface should return a byte array for the Flight request.
+
+
+Configuration
```
Comment (steveburnett) [inline presto-docs/src/main/sphinx/connector/base-arrow-flight.rst:31]: This Configuration section is really clean and well written. No edits, just wanted to say this looks good.

**Review by steveburnett — CHANGES_REQUESTED** [review]
Thanks for the documentation! A few suggestions and comments, looks good overall.

**Comment by lithin92** [comment]
@steveburnett This is just a test PR to check my new CI job . I will close this soon

**Pull request closed without merging**